tests/test_matrix_utils_relay.py

```
@@ -0,0 +1,2607 @@
+        Returns:
+            str: The display name "user".
+        """
+        return "user"
+
+    dummy_queue = MagicMock()
+    dummy_queue.get_queue_size.return_value = 0
+
+    real_loop = asyncio.get_running_loop()
+
+    class DummyLoop:
+        def __init__(self, loop):
```

Ruff (0.15.9)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)